api-report/counter.api.md

```diff
@@ -0,0 +1,49 @@
+## API Report File for "@fluidframework/counter"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { IChannelFactory } from '@fluidframework/datastore-definitions';
+import { IChannelStorageService } from '@fluidframework/datastore-definitions';
+import { IFluidDataStoreRuntime } from '@fluidframework/datastore-definitions';
+import { IFluidSerializer } from '@fluidframework/core-interfaces';
+import { ISequencedDocumentMessage } from '@fluidframework/protocol-definitions';
+import { ISharedObject } from '@fluidframework/shared-object-base';
+import { ISharedObjectEvents } from '@fluidframework/shared-object-base';
+import { ITree } from '@fluidframework/protocol-definitions';
+import { SharedObject } from '@fluidframework/shared-object-base';
+
+// @public
+export interface ISharedCounter extends ISharedObject<ISharedCounterEvents> {
+    increment(incrementAmount: number): void;
+    value: number;
+}
+
+// @public (undocumented)
+export interface ISharedCounterEvents extends ISharedObjectEvents {
+    // (undocumented)
+    (event: "incremented", listener: (incrementAmount: number, newValue: number) => void): any;
+}
+
+// @public
+export class SharedCounter extends SharedObject<ISharedCounterEvents> implements ISharedCounter {
+    // (undocumented)
+    protected applyStashedOp(): void;
+    static create(runtime: IFluidDataStoreRuntime, id?: string): SharedCounter;
```

Comment:
Shouldn't this return an `ISharedCounter` instead of the class?

Reply:
I'd probably align it with our decision in #6399 - other DDS classes `static create()`s return the class currently.